pkg/yurthub/kubernetes/meta/restmapper.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2021 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	normaljson "encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/openyurtio/openyurt/pkg/yurthub/storage"
+	"github.com/openyurtio/openyurt/pkg/yurthub/storage/disk"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog"
+)
+
+const (
+	CacheDynamicRESTMapperKey = "_internal/restmapper/cache-crd-restmapper.conf"
+	SepForGVR                 = "/"
+)
+
+var (
+	// unsafeDefaultRESTMapper is used to store the mapping relationship between GVK and GVR in scheme
+	unsafeDefaultRESTMapper = NewDefaultRESTMapperFromScheme()
```

Comment:
var `unsafeDefaultRESTMapper` has the same name as struct field `RESTMapperManager. unsafeDefaultRESTMapper`, we need different these two definitions.

Reply:
The code is updated, the global variable name is changed to `unsafeSchemeRESTMapper`.